.github/workflows/draft-pdf.yml

```
@@ -0,0 +1,28 @@
+name: Draft PDF
+on:
+    push:
+      paths:
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)